tests/integration-tests/tests/update/test_update.py

```diff
@@ -1052,3 +1069,90 @@ def _test_shared_storages_mount_on_headnode(
         bucket_name,
         headnode_only=True,
     )
+
+
+def _test_shared_storage_rollback(
+    cluster,
+    existing_ebs_volume_id,
+    existing_ebs_mount_dir,
+    bucket_name,
+    new_ebs_mount_dir,
+    new_lustre_mount_dir,
+    new_efs_mount_dir,
+    remote_command_executor,
+    pcluster_config_reader,
+    scheduler_commands,
+    region,
+):
+    # update cluster with adding non-existing ebs and skip validator
+    problematic_volume_id = "vol-00000000000000000"
+    problematic_ebs_mount_dir = "/problematic_ebs_mount_dir"
+    problematic_cluster_config = pcluster_config_reader(
+        config_file="pcluster.config.update_rollback.yaml",
+        volume_id=existing_ebs_volume_id,
+        problematic_volume_id=problematic_volume_id,
+        problematic_ebs_mount_dir=problematic_ebs_mount_dir,
+        existing_ebs_mount_dir=existing_ebs_mount_dir,
+        bucket_name=bucket_name,
+        new_ebs_mount_dir=new_ebs_mount_dir,
+        new_lustre_mount_dir=new_lustre_mount_dir,
+        new_efs_mount_dir=new_efs_mount_dir,
+    )
+    # remove logs from chef-client log in order to test cluster rollback behavior
+    # remote_command_executor.run_remote_command("sudo truncate -s 0 /var/log/chef-client.log")
+    response = cluster.update(
+        str(problematic_cluster_config),
```

Comment:
No, it works with or without str()

Reply:
Thank you. If it's not required, let's remove it then.

Reply:
`str()` in cluster.update(str(config_path)) exists everywhere in the integration tests. No sure if it is here to prevent some error or not, I am not in favor of removing it here for this specified test. If we want to refactor it, we can do it for all the integration test.

Reply:
I agree.